<launch>

	<arg name="publish_tf" default="False"/>
	<node name = "diff_odom" pkg = "roboteq_motor_controller_driver" type = "diff_odom_node" output = "screen">
		<param name="publish_tf" type="bool" value="$(arg publish_tf)"/>
		<!-- <remap from="cmd_vel" to="not_used_cmd_vel"/> -->
	</node>
</launch>
